python/dgl/graphbolt/minibatch.py

```diff
@@ -395,7 +395,11 @@ def blocks(self):
                             (
                                 v.indptr,
                                 v.indices,
-                                torch.tensor([]),
+                                torch.arange(
+                                    0,
+                                    len(v.indices),
+                                    device=v.indptr.device,
```

Comment:
use length of `indices` while device is retrieved from `indptr`? though `indptr` and `indices` are supposed to be on same device?

Reply:
`indptr` and `indices` are supposed to be on the same device because they are all in the `CSCFormatBase`. Should I change it to `indices`？

Reply:
the 3rd argument is supposed to be edge ID? https://docs.dgl.ai/en/latest/generated/dgl.create_block.html#dgl.create_block

Reply:
Yes, I have tried to use `original_edge_ids` before, however it ran segmentation fault.

Reply:
to be consistent, it's better to keep them same. retrieve device type from `indices` too.

Reply:
could we replace `len(v.indices)` with `v.indptr[-1]` for better performance?

Reply:
> to be consistent, it's better to keep them same. retrieve device type from `indices` too.

Due to use indptr[-1], I still retrieve device type from `indptr`.